asreview/webapp/src/ProjectComponents/SetupComponents/DataComponents/AddDataset.js

```diff
@@ -75,7 +75,8 @@ const AddDataset = (props) => {
               <code>.txt</code>) and tabular datasets (<code>.csv</code>,{" "}
               <code>.tab</code>, <code>.tsv</code>, <code>.xlsx</code>). The
               selected dataset should contain the title and abstract of each
-              record.{" "}
+              record. It is highly recommended to add a dataset without{" "}
+              duplicate records.{" "}
```

Comment:
```suggestion
              record. To optimally benefit from the performance of the active
              learning model, it is highly recommended to add a dataset 
              without duplicate records.{" "}
```

Reply:
I propose to replace:

"The selected dataset should contain the title and abstract of each record. It is highly recommended to add a dataset without duplicate records."

with:

"To optimally benefit from the performance of the active learning model it is essential to upload high-quality data: provide unique records (after de-duplication) and avoid missing information."

only 4 more words

Reply:
Before we can think about the text, we need to come to a consensus on what is necessary. 

> @J535D165 General comment (also for the other reviewers): I think it should be a recommendation.

It is essential to add a dataset that contains:
- [x] title
- [ ] abstract
- [ ] no duplicate records

Reply:
Duplicate records can be intentional (ping @bmkramer) so we should not use the phrase "no duplicate records".

Reply:
So adding a dataset without duplicate records is highly recommended but not essential.

Reply:
> upload high-quality data: provide unique records (after de-duplication) and avoid missing information

This is not concise according to the writing principle. https://www.material.io/design/communication/writing.html#principles

"High-quality data" is a lengthy description. "Missing information" is an implication.
“Unique records" is also an implication so it has "after de-duplication" attached to it. Then it becomes lengthy.

I still suggest that half a sentence explaining why it is highly recommended to add a dataset without duplicate records is sufficient.